main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -313,17 +315,20 @@ public void update(long bytesRead, long contentLength, int itemCount) {
                                 throw new Exception("No content found in " + url.toString());
                             }
                             InputStream stream2 = entity.getContent();
-                            String encoding = null;
-                            if (entity.getContentEncoding() != null) {
-                                encoding = entity.getContentEncoding().getValue();
-                            }
-                            JSONUtilities.safePut(fileRecord, "declaredEncoding", encoding);
```

Comment:
Is it never useful to read the encoding from `entity.getContentEncoding()`? Is the `getCharset()` call you replaced it with always preferable (or should there be a fallback mechanism)?

Reply:
Never useful and not relevant. Content-Encoding specifies compression. In the case of your example, it will be `gzip`, but then get written to `null` when the request handler decompresses the content.

Reply:
@tfmorris that's useful as a 1 line code comment there I would say. Disambiguating the `Encoding` term on the spot.

Reply:
This conversation is about deleted code. That's why it's highlighted in red.

Reply:
Oops, overlooked. Ok, well then not *there*